### ISSUE TITLE
[Bug] stocks/sia/vis only working 10% of the time due to StockAnalysis

### ISSUE THREAD

**Issue by JerBouma** [OpenBB-finance/OpenBBTerminal#1514]
**Describe the bug**
It seems StockAnalysis.com added an additional layer of protection.  Running this command `python terminal.py stocks/load aal/sia/clear country/vis re` works for the first/second time but then it starts blocking you resulting in no data found. When looking at the `soup` I noticed a Cloudflare page popping up, it seems to be some form of ant-bot measures deployed.

**To Reproduce**
Running this command a couple of times `python terminal.py stocks/load aal/sia/clear country/vis re`. You can see that it previously worked from #1507.

**Comment by colin99d** [comment]
If we are getting a Cloudflare pop up that is unfortunately pretty hard to work around. I was looking into trying to hit their API directly, but it appears that they encrypt the data (the only data I saw was stuff we did not want and then a lot of data that was unintelligible characters). StockAnalysis is the only free service that provides enough data to make a DCF reasonably possible. I think at this point we need to either have Didier talk with the guy and see why he is trying to stop us from getting data, or try to come up with other options.
